cmd/bundle.rb

```diff
@@ -162,22 +162,21 @@ def bundle
       end
     rescue SystemExit => e
       Homebrew.failed = true unless e.success?
-      puts "Kernel.exit" if args.debug?
+      odebug "Kernel.exit" if args.debug?
```

Comment:
```suggestion
      odebug "Kernel.exit"
```

may work as expected, worth a try.